var/spack/repos/builtin/packages/pumi/package.py

```diff
@@ -0,0 +1,64 @@
+##############################################################################
+# Copyright (c) 2013-2016, Lawrence Livermore National Security, LLC.
+# Produced at the Lawrence Livermore National Laboratory.
+#
+# This file is part of Spack.
+# Created by Todd Gamblin, tgamblin@llnl.gov, All rights reserved.
+# LLNL-CODE-647188
+#
+# For details, see https://github.com/llnl/spack
+# Please also see the LICENSE file for our notice and the LGPL.
+#
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU Lesser General Public License (as
+# published by the Free Software Foundation) version 2.1, February 1999.
+#
+# This program is distributed in the hope that it will be useful, but
+# WITHOUT ANY WARRANTY; without even the IMPLIED WARRANTY OF
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the terms and
+# conditions of the GNU Lesser General Public License for more details.
+#
+# You should have received a copy of the GNU Lesser General Public
+# License along with this program; if not, write to the Free Software
+# Foundation, Inc., 59 Temple Place, Suite 330, Boston, MA 02111-1307 USA
+##############################################################################
+from spack import *
+import sys
+
+
+class Pumi(CMakePackage):
+    """SCOREC RPI's Parallel Unstructured Mesh Infrastructure (PUMI).
+       An efficient distributed mesh data structure and methods to support
+       parallel adaptive analysis including general mesh-based operations,
+       such as mesh entity creation/deletion, adjacency and geometric
+       classification, iterators, arbitrary (field) data attachable to mesh
+       entities, efficient communication involving entities duplicated
+       across multiple tasks, migration of mesh entities between tasks,
+       and dynamic load balancing."""
+
+    homepage = "https://www.scorec.rpi.edu/pumi"
+    url      = "https://github.com/SCOREC/core.git"
+
+    version('master', git='https://github.com/SCOREC/core.git',
+        branch='master')
```

Comment:
> will fix

thanks.

p.s. IMO you do NOT really have to reply to each comment that you are fixing it. When you push changes where this line is modified, then the whole comment will be collapsed/closed. It is then understood that you changed this line according to the comment.